eWallet/eWallet/settings.py

```diff
@@ -134,4 +134,13 @@
   cloud_name = "db3fj4gb9", 
   api_key = "429755737912151", 
   api_secret = "vna3cvnu6_levKhdW1HZpPF81lM" 
-)
\ No newline at end of file
+)
+
+#Email settings
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
+EMAIL_USE_TLS = True
+EMAIL_HOST_USER = 'hafsatariq513@gmail.com'
+EMAIL_HOST_PASSWORD = 'kzwtdqvgbxrbqvdt'
+DEFAULT_FROM_EMAIL = 'hafsatariq513@gmail.com'
```

Comment:
Create an env file and add all these setting there. And add that env in gitignore.